Abilities/Punch.tscn

```diff
@@ -73,7 +73,7 @@ func _on_Collider_body_exited( body ):
 		body.setHighlight(false)
 "
 
-[node name="Punch" type="Node2D" index="0"]
```

Comment:
If possible try not to commit these weird auto changes that godot does for no reason, not sure if your git viewer shows it clearly or not.